src/core/main.cpp

```diff
@@ -528,6 +528,57 @@ int main( int argc, char * * argv )
 		}
 	}
 
+	// Test file argument before continuing
+	QFileInfo fileInfoLoad( fileToLoad );
+	QFileInfo fileInfoImport( fileToImport );
+	if( !fileToLoad.isEmpty() )
```

Comment:
The above code comment from April still remains unanswered and you have a commented line that should be removed prior to merging.